test/conftest.py

```diff
@@ -1,13 +1,47 @@
+import socket
 import pytest
+import subprocess
+import time
+import os
 from seam import Seam
-import random
-import string
+from contextlib import contextmanager
+
+
+def find_free_port():
```

Comment:
Mainly because we already use this and people are familiar with it https://www.npmjs.com/package/get-port so we can just use one name across our projects to refer to this function.

```suggestion
def get_port():
```

Reply:
447e37de09bf11aaff54eed1b386758e90f28a2b